src/Symfony/Component/Messenger/Bridge/Redis/Tests/Transport/ConnectionTest.php

```diff
@@ -191,17 +191,6 @@ public function testAuthFromOptionsAndDsn()
         Connection::fromDsn('redis://password1@localhost/queue', ['auth' => 'password2'], $redis);
     }
 
-    public function testAuthAsUserInDsn()
-    {
-        $redis = $this->createMock(\Redis::class);
-
-        $redis->expects($this->exactly(1))->method('auth')
-            ->with('password')
-            ->willReturn(true);
-
-        Connection::fromDsn('redis://password:localhost/queue', [], $redis);
```

Comment:
Please change this to: 

```
Connection::fromDsn('redis://password@localhost/queue', [], $redis);
```

Reply:
If I do, we're going to have a duplicate :) https://github.com/symfony/symfony/blob/ccc5513f547654ba0b37626d7f54fbd6a6086893/src/Symfony/Component/Messenger/Bridge/Redis/Tests/Transport/ConnectionTest.php#L161-L170

Reply:
Oh, Okey. I didn't check that.